frontend/src/libs/maps/icon-name-to-plain-svg.tsx

```diff
@@ -0,0 +1,21 @@
+import { ReactComponent as ArrowIcon } from '#assets/icons/arrow.svg';
+import { ReactComponent as AvatarIcon } from '#assets/icons/avatar.svg';
+import { ReactComponent as NotificationIcon } from '#assets/icons/notification.svg';
+import { ReactComponent as PlusIcon } from '#assets/icons/plus.svg';
+import { ReactComponent as SignOutIcon } from '#assets/icons/sign-out.svg';
+import { ReactComponent as SubscriptionIcon } from '#assets/icons/subscription.svg';
+import { type IconNames } from '#libs/enums/enums.js';
+
+const iconNameToPlainSvgMap: Record<
+  IconNames,
+  React.FunctionComponent<React.SVGProps<SVGSVGElement>>
+> = {
+  'arrow': ArrowIcon,
+  'avatar': AvatarIcon,
+  'plus': PlusIcon,
+  'sign-out': SignOutIcon,
+  'notification': NotificationIcon,
+  'subscription': SubscriptionIcon,
```

Comment:
Do we need quotes here?

Reply:
I think yes, because we can't use 'sign-out' without quotes

Reply:
yeah, but quotes are only needed for 'sign-out', aren't they?

Reply:
so it should be all without quotes and only one with quotes? May look messy doesn't it?

Reply:
This map should be in icon package